src/org/labkey/test/tests/query/QueryLookupTest.java

```diff
@@ -0,0 +1,148 @@
+package org.labkey.test.tests.query;
+
+import org.assertj.core.api.Assertions;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.pages.list.EditListDefinitionPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.list.IntListDefinition;
+
+import org.labkey.test.params.list.VarListDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.EscapeUtil;
+import org.labkey.test.util.data.TestDataUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+
+@Category({Daily.class})
+public class QueryLookupTest extends BaseWebDriverTest
+{
+    private static final String PROJECT_NAME = "QueryLookupTest" + TRICKY_CHARACTERS_FOR_PROJECT_NAMES;
+    private static final String LIST_NAME = "l&ist q";
+
+    private static final FieldDefinition NAME_COLUMN =
+            new FieldDefinition("Name", FieldDefinition.ColumnType.String);
+    private static final FieldDefinition TSHIRT_COLUMN =
+            new FieldDefinition("TShirt", FieldDefinition.ColumnType.String);
```

Comment:
`FieldDefinition` is not immutable, so isn't really appropriate to define as a constant. Consider using the shiny new `FieldInfo`.
```suggestion
    private static final FieldInfo NAME_COLUMN =
            new FieldInfo("Name", FieldDefinition.ColumnType.String);
    private static final FieldInfo TSHIRT_COLUMN =
            new FieldInfo("TShirt", FieldDefinition.ColumnType.String);
```
Then you can use `getFieldDefinition()` to create the list (e.g. `NAME_COLUMN.getFieldDefinition()`).